ocs_ci/deployment/hosted_cluster.py

```diff
@@ -376,18 +384,35 @@ def verify_hosted_ocp_clusters_from_provider(self):
             )
             return False
 
-    def download_hosted_clusters_kubeconfig_files(self):
+    def download_hosted_clusters_kubeconfig_files(self, cluster_names_paths_dict=None):
         """
-        Get HyperShift hosted cluster kubeconfig for multiple clusters
+        Get HyperShift hosted cluster kubeconfig for multiple clusters.
+        Provided cluster_names_paths_dict will always be a default source of cluster names and paths
+
+        Args:
+            cluster_names_paths_dict (dict): Optional argument. Function will download all kubeconfigs to specified in
+            configuration folders or download specific clusters kubeconfig to specified in arguments folder
```

Comment:
This sentence seems to require some update to make it clear.

Reply:
done